### ISSUE TITLE
Bad allocation error while runnins simulations

### ISSUE THREAD

**Issue by pvpoli** [diyabc/diyabcGUI#76]
Hello,
I am getting a bad allocation error while running simulations in the training data. No other informative (for me) errors are shown. The computer has enough memory available. What am I doing wrong?
Thank you for your help. 
Sorry if this is not the right place to post this issue. 
[diyabc_run_call.log](https://github.com/diyabc/diyabcGUI/files/5983922/diyabc_run_call.log)

**Comment by gdurif** [comment]
Hi,
You posted at the right place. Sorry for the bug.
Could you give me more details about the scenario, parameter values and type of data you were trying to simulate ?
Thanks in advance
Best,

**Comment by pvpoli** [comment]
Hi, 
thank you for your quick answer. 
Actually, after quitting diyabc, and closing and reopening firefox, all went well. I think perhaps it was firefox that did not have enough available memory? I did yesterday have around 8 tabs open at the time I tried to run diyabc.
Anyway, I was trying 5 scenarios of population expansion and splitting based on microsatellite loci. All mutation parameters were kept as default, and historical parameters were adjusted so that times of populations split and merge made sense. 
Let me know if you need any particular file from that run, or any other information. I'll make some runs today and see if I get the same error.
Best,

**Comment by gdurif** [comment]
Hi, a new version of the GUI is at least available on branch `refactor_app`, see PR #96 

If you want to try it first hand, you can install the R package and run the app with:
```R
devtools::install_github("diyabc/diyabcGUI", ref = "refactor_app")
dl_all_latest_bin()
diyabc()
```

It will be merged tomorrow in `master` and a new version of the beta release will be published.
